app/viewmodels/cya/CYAEntryDetailsSummaryListHelper.scala

```diff
@@ -119,7 +119,10 @@ trait CYAEntryDetailsSummaryListHelper {
       createRow(
         keyText = Text(messages("cya.oneCustomsProcedureCode")),
         valueContent = Text(oneCustomsProcedureCode),
-        action = Some(ActionItem("Url", Text(messages("cya.change"))))
+        action = Some(ActionItemHelper.createChangeActionItem(
+          controllers.routes.OneCustomsProcedureCodeController.onLoad().url,
+          messages("cya.acceptanceDate.change")
```

Comment:
should this be a different message to Acceptance date? Also below.

Reply:
Updated